### PR TITLE
MVP Development

### PR DESCRIPTION
**Pull request by nawaidshamim**
Updated models and endpoints as per MVP requirements. This is has been deployed as version 5 on elite-firefly-737 project.

### CODE REVIEW

**Comment by nawaidshamim** [comment]
This is has been deployed as version 5 on elite-firefly-737 project.

File: models/account.py [hunk 1]
```diff
@@ -55,3 +53,19 @@ def IdSet(self, value):
     def id(self):
         if self.key is not None:
             return self.key.string_id()
+
+
+    # fields below are not part of MVP and maybe removed
```
Comment (patt0) [inline models/account.py:75]: **fields that will are renamed**
gplus_id -> social_googleplus
so_id -> social_stackoverflow

**fields that seem necessary**
product_group

**fields that support maps & flags & profile**
ctry_filename
pg_filename
geocode
pic_url

**Soft Delete**
deleted

Comment (nawaidshamim) [inline models/account.py:75]: Added these to [README.md](README.md)

**Comment by patt0** [comment]
These changes do not need to go in Master anytime soon. We can reopen this anytime.

**Pull request closed without merging**